be/src/storage/update_manager.h

```diff
@@ -38,6 +38,7 @@ class RowsetUpdateState;
 class RowsetColumnUpdateState;
 class Tablet;
 class PersistentIndexCompactionManager;
+using TabletAndScore = std::pair<int64_t, double>;
```

Comment:
Don't define `TabletAndScore` twice, you can define it in `persistent_index_compaction_manager.h`